straxen/plugins/event_info.py

```diff
@@ -54,3 +57,57 @@ class EventInfo1T(strax.MergeOnlyPlugin):
     provides = 'event_info'
     save_when = strax.SaveWhen.ALWAYS
     __version__ = '0.0.1'
+
+
+@export
+class EventInfoVetos(strax.Plugin):
+    """
+    Plugin which combines event_info with the tagged peaks information
+    from muon- and neutron-veto.
+    """
+    __version__ = '0.0.0'
+    depends_on = ('event_info', 'peak_veto_tags')

```

Comment:
Yes event basics would be fine too. This relates to question you were asking above. Yes I think in linked mode we should compute these data types by default. For this reason I made the dependency on `event_info` and not `event_basics`. I thought it might be easier to process data in the following order:

1. TPC
2. mv/nv-events
3. TPC and vetos joined data types.

But I am open to suggestions as you have much more experience here than I have.

Reply:
Generally I think it's better to depend on low-level stuff then high level stuff when it's not needed.

I'm not sure how easy it's going to be to compute this, I guess we will have to try and see